requirements.txt

```diff
@@ -0,0 +1,3 @@
+setuptools
+zc.buildout
+etk.docking
```

Comment:
The `etk.docking` dependency can be dropped.

It would be nice to have a build step that does an actual buildout build, so we can validate it remains working.